### DISCUSSION TITLE
Why is there no Branch object in the GraphQL documentation?

### DISCUSSION THREAD

**Discussion by dlinch** [community/community#52195] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

### Body

I was reviewing a potential project I wanted to work on that would involve doing a little housekeeping in our Github repo, specifically around branches. While digging through the GraphQL documentation I couldn't find any reference to a Branch object, not like the representation of a `GET /branch/:id` the REST API provides. I can only see references to BranchProtection objects.

**Comment by jdecode** [comment]
`defaultBranchRef`  points to the default branch

`ref` points to the branch that you provide (as a variable to the `ref` method)
